examples/hello_world/CMakeLists.txt

```diff
@@ -19,7 +19,10 @@ if(MSVC)
     )
 endif()
 
+find_package(spdlog REQUIRED)
```

Comment:
Let's implement our own simple logger. It's less of a hassle. I don't think it needs to be anything fancy - formatting can be done with `vsnprintf`, with a thread-local buffer, with printing to some configurable sink (but `stdout`/`stderr` should be sufficient for now).